src/CoreText/CTStringAttributes.cs

```diff
@@ -193,6 +197,15 @@ public CTStringAttributes (NSDictionary dictionary)
 			set {Adapter.SetNativeValue (Dictionary, CTStringAttributeKey.ForegroundColor, value);}
 		}
 
+		[iOS (10,0)][Mac(10,12)]
+		public CGColor BackgroundColor {
+			get {
+				var h = CFDictionary.GetValue (Dictionary.Handle, CTStringAttributeKey.BackgroundColor.Handle);
```

Comment:
note: unmanaged bindings needs unit tests.

In this case an older version of iOS will likely return `null` for `CTStringAttributeKey.BackgroundColor` so an `NullReferenceException` will be thrown here. Using `?.` (simpler) would likely solve this (and return `null`).